.github/workflows/push_build_dev.yaml

```diff
@@ -0,0 +1,106 @@
+name: "[Push] Build Dev"
+
+on:
+  workflow_dispatch:
+  push:
+      branches:
+        - master
+
+env:
+  DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+  DOCKER_ORGANIZATION: ${{ secrets.DISCORD_WEBHOOK || 'cloudforet }}
```

Comment:
-1
set as a environment value, you don't need to switch at line 64

Reply:
in a github environment, set DOCKER_ORGANIZATION, DOCKER_USERNAME, DOCKER_PASSWORD. in this case, you don't need to switch at line 64.

Reply:
Some people may be DOCKER_ORGANIZATION != DOCKER_USERNAME, How about set separate theree environments, secrets.DOCKER_ORGANIZATION, secrets.DOCKER_USERNAME, secrets.DOCKER_PASSWORD?

Reply:
I update env variables.